core/src/main/java/de/danoeh/antennapod/core/service/playback/RemotePSMP.java

```diff
@@ -546,13 +546,13 @@ public void endPlayback(boolean wasSkipped, boolean switchingPlayers) {
         } catch (TransientNetworkDisconnectionException | NoConnectionException e) {
             Log.e(TAG, "Could not determine if media is playing", e);
         }
-        if (wasSkipped) {
-            try {
-                castMgr.stop();
-            } catch (CastException | TransientNetworkDisconnectionException | NoConnectionException e) {
-                Log.e(TAG, "Could not stop remote playback when skipping", e);
-            }
-        }
+//        if (wasSkipped) {
```

Comment:
As a general answer, I commented pieces before testing the effects of doing it and to easily revert it in case the results are not what I expected. Then what happens is that I forget to remove the comments when committing. I understand that's okay when coding to myself, not so ok when collaborating, so I'll get rid of them.

In this particular case I initially wanted to give almost immediate feedback to the user once we skip an episode, but as it turns out the communications with chromecast tend to be pretty unreliable, as you cannot assume that 2 commands that are sent will be processed in the order they were issued, so we should minimize the amount of instructions that we send.
On the other hand, I might need to recover this block, possibly in a different part of the code, because I believe playback doesn't stop if we happen to skip the last episode in the queue, so in a way it serves as a reminder, although a TODO would be more appropriate.